data/json/professions.json

```diff
@@ -4473,5 +4479,58 @@
       }
     },
     "flags": [ "SCEN_ONLY" ]
+  },
+  {
+    "type": "profession",
+    "id": "ninja",
+    "name": "Ninja",
+    "description": "You always dreamed with being a ninja, so you practiced your movements, dressed all in black and carried some common ninja weapons with you.  Now, after much practice, you can finally call yourself a real ninja.",
+    "points": 5,
+    "skills": [
+      { "level": 2, "name": "melee" },
+      { "level": 2, "name": "cutting" },
+      { "level": 3, "name": "dodge" },
+      { "level": 1, "name": "swimming" }
+    ],
+    "items": {
+      "both": {
+        "items": [ "hood_ninja", "jacket_ninja", "trousers_ninja", "tabi_dress", "geta", "gloves_wraps", "bellywrap" ],
+        "entries": [
+          { "item": "katana", "container-item": "scabbard" },
+          { "item": "grenadebandolier", "contents-group": "ninja_grenade_pouch" }
+        ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "chestwrap", "panties" ],
+      "traits": [ "LIGHTSTEP" ]
```

Comment:
```suggestion
    ],
     "traits": [ "LIGHTSTEP" ],
    "items": {
      "both": {
        "items": [ "hood_ninja", "jacket_ninja", "trousers_ninja", "tabi_dress", "geta", "gloves_wraps", "bellywrap" ],
        "entries": [
          { "item": "katana", "container-item": "scabbard" },
          { "item": "grenadebandolier", "contents-group": "ninja_grenade_pouch" }
        ]
      },
      "male": [ "boxer_shorts" ],
      "female": [ "chestwrap", "panties" ]
```

Reply:
Agh, I had already fixed (forgetting commas...) in my local branch, but thanks!

Reply:
Nevermind, I see that you fixed it just now :P